src/redux/search/reducer.js

```diff
@@ -0,0 +1,31 @@
+import { SEARCH_START, SEARCH_SUCCESS, SEARCH_FAILED } from './actionCreater';
+
+const initState = {
+  isloading: false,
+  result: [],
+  error: '',
+};
+
+export default (state = initState, action) => {
+  switch (action.type) {
+    case SEARCH_START:
+      return {
+        ...state,
```

Comment:
don't we need to reset the errors and the result when the suer starts a new search?